org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/thingTranslator.js

```diff
@@ -120,69 +128,95 @@ class WinkThermostat {
         console.log('Javascript and Wink Helper initialized : ');
     }
 
-    // exports for the OCF schema
+    // exports for the entire schema object
 
     // Queries the entire state of the thermostat
     // and returns an object that maps to the json schema org.opent2t.sample.thermostat.superpopular
     getThermostatResURI() {
         return winkHelper.getDeviceDetailsAsync(deviceType, deviceId)
             .then((response) => {
-                return winkSchemaToOcfSchema(response.data);
+                return deviceSchemaToTranslatorSchema(response.data);
             });
     }
 
-    // Updates the current state of the thermostat with the contents of value
-    // value is an object that maps to the json schema org.opent2t.sample.thermostat.superpopular
+    // Updates the current state of the thermostat with the contents of postPayload
+    // postPayload is an object that maps to the json schema org.opent2t.sample.thermostat.superpopular
     //
     // In addition, returns the updated state (see sample in RAML)
-    postThermostatResURI(value) {
-        var putPayload = ocfSchemaToWinkSchema(value);
+    postThermostatResURI(postPayload) {
+        console.log('postThermostatResURI called with payload: ' + JSON.stringify(postPayload));
```

Comment:
> JSON.stringify(postPayload));

Since this is for a human-readable log, consider using the 'space' parameter for JSON.stringify to make the output more easily readable (see https://msdn.microsoft.com/library/cc836459(v=vs.94).aspx).